packages/subgraph/schema.graphql

```diff
@@ -21,20 +21,20 @@ type Balance @entity {
 
 # ACL
 
-type Role @entity {
-  id: ID! # where + role
+type ContractPermissionID @entity {
+  id: ID! # where + permissionID
   dao: Dao!
   where: Bytes!
-  role: Bytes!
-  frozen: Boolean!
+  permissionID: Bytes!
+  immutable: Boolean!
 }
 
 type Permission @entity {
   "no need to store granted as we can delete permission when revoked"
-  id: ID! # where + role + who
+  id: ID! # where + permissionID + who
   dao: Dao!
   where: Bytes!
-  role: Role!
+  contractPermissionID: ContractPermissionID!
```

Comment:
why not simply `permissionId`?

Reply:
This is some different subgraph entity: 

https://github.com/aragon/core/blob/4f2bb033be78ab40f80194dadc81581db4434b63/packages/subgraph/schema.graphql#L24-L41

The subgraph should be documented and the naming should be refactored as well.

Reply:
not fully understand the concept. but prefer permissionId.

Reply:
Those are changes to the subgraph that are not in the scope of this PR.